lib/lettering/font.py

```diff
@@ -113,6 +113,14 @@ def _check_variants(self):
     min_scale = font_metadata('min_scale', 1.0)
     max_scale = font_metadata('max_scale', 1.0)
 
+    # Version 2 : For use values ​​from SVG Font, exemple <font horiz-adv-x="45" ...  <glyph .... horiz-adv-x="49" glyph-name="A" /> ... <hkern ... k="3"g1="A" g2="B" /> .... />
+    version = font_metadata('version', '') # Exemple font.json : "version":"2"
+    kerning_pairs_px = font_metadata('kerning_pairs_px', {}) #Exemple font.json : "kerning_pairs_px": {"AB":3},
+    horiz_adv_x = font_metadata('horiz_adv_x', {}) # Exemple font.json : "horiz_adv_x": {"A":49},
+    horiz_adv_x_default = font_metadata('horiz_adv_x_default', 0) # Exemple font.json : "horiz_adv_x_default" : 45,
+    if version == "2":
```

Comment:
This won't work the way you're hoping it will, unfortunately. :(  `version` is a `property` object, so it will definitely never `== "2"`.

Instead, you'll need to check the `version` where you're using `word_spacing`.

Reply:
Definitely I'm angry with the version variable. It is again a mistake of inattention.